warrior/Actions/SnmpActions/common_snmp_actions.py

```diff
@@ -96,7 +100,7 @@ def snmp_get(self, snmp_ver, system_name, mib_name=None,
                               e.g. privProtocol="1,3,6,1,6,3,10,1,2,2"
             14.custom_mib_paths: User can provide multiple MIB source path seperated by comma (',')
                       Source path can be url or just absolute directory path. Refer bellow example.
-                      e.g. 'https://github.com/warriorframework/warriorframework, /data/users/username/MIBS/FNC'.
+                      e.g. 'http://rtx-swtl-git.fnc.net.local/projects/TYREPO/repos/fujitsu_base_yang_repo/browse/src/util/snmp/@mib@, /data/users/sbhattac/MIBS/FNC'.
```

Comment:
@sbhattach please don't put fujitsu related information any where in OSS code as it is available for everyone. Please revert or use fujitsu independent web address

Reply:
Corrected.